packages/components/src/components/primitives/Tooltip/Tooltip.tsx

```diff
@@ -1,10 +1,14 @@
-import React from 'react';
-import ReactTooltip from 'react-tooltip';
-import colors from '../../../../lib/styles/colors.js';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { usePopper } from 'react-popper';
+import { getStyles } from './Tooltip.styles';
 
 export const TOOLTIP_DEFAULT_TEST_ID = 'tooltip';
 
-const defaultColor = colors['fill-primary-900'];
+export enum BackgroundColor {
```

Comment:
Rename to `TooltipBackgroundColor` to be consistent?

Reply:
done!